features/step_definitions/transaction_steps.rb

```diff
@@ -22,3 +22,10 @@
 When /^the buyer makes (\d+|a) service transactions? with:$/ do |count, table|
   step %(buyer "#{@buyer.name}" makes #{count} service transactions with:), table
 end
+
+Given "the backend will respond to a utilization request for application {string} with:" do |application_name, json|
+  application = Cinstance.by_name(application_name).first
+  TestHelpers::Backend::MockCore.stubs.get("/internal/services/#{application.service_id}/applications/#{application.application_id}/utilization/") do
+    [ 200, {'content-type'=>'application/json'}, json ]
```

Comment:
After a hell of hours of pure suffering, I found this seems to be the one and only way to stub requests to Backend from tests, so I wonder why [this](https://github.com/3scale/porta/blob/f6e98d1d8a66f28d13a91ebba8767e12814509ae/features/old/applications/buyers/referrer_filters/buyer_side_multiple_applications.feature#L42) works...

Reply:
That makes sense